project-base/app/tests/FrontendApiBundle/Functional/Product/ProductSellingDeniedOnDomainTest.php

```diff
@@ -46,7 +40,7 @@ public function testSellingDeniedOnDomain(): void
 
         $this->dispatchFakeKernelResponseEventToTriggerImmediateRecalculations();
```

Comment:
is this line still needed? Ah, I see it is replaced in the WIP commit with `handleDispatchedRecalculationMessages`. So now, you call it twice in this test

Reply:
thanks, removed one call